fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/FluidView.java

```diff
@@ -0,0 +1,125 @@
+package net.fabricmc.fabric.api.fluids;
+
+import java.util.Collections;
+import java.util.IdentityHashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Iterators;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Box;
+import net.minecraft.util.math.Direction;
+import net.minecraft.world.World;
+
+import net.fabricmc.fabric.api.fluids.containers.FluidContainer;
+import net.fabricmc.fabric.api.fluids.containers.SimpleFluidContainer;
+import net.fabricmc.fabric.api.fluids.containers.volume.ImmutableFluidVolume;
+import net.fabricmc.fabric.api.fluids.items.ItemFluidContainer;
+import net.fabricmc.fabric.api.fluids.items.ItemSink;
+import net.fabricmc.fabric.api.fluids.math.Drops;
+import net.fabricmc.fabric.api.fluids.minecraft.items.BottleItemFluidContainer;
+import net.fabricmc.fabric.api.fluids.minecraft.items.BowlItemFluidContainer;
+import net.fabricmc.fabric.api.fluids.world.SidedFluidContainer;
+
+/**
+ * A util class for accessing fluid containers
+ * <p>
+ * Introduction:
+ *
+ * @see Drops
+ */
+public class FluidView {
+	public static final Map<Item, ItemFluidContainer> FLUID_CONTAINERS = new IdentityHashMap<>();
+
+	static {
+		FLUID_CONTAINERS.put(Items.BOWL, ((waste, stack) -> new BowlItemFluidContainer(stack, waste)));
+		FLUID_CONTAINERS.put(Items.DRAGON_BREATH, ((waste, stack) -> new BottleItemFluidContainer(stack, waste)));
```

Comment:
We really should have all these vanilla constants in one place, I dunno mod init?

Reply:
that is gone